packages/common/src/utils/index.ts

```diff
@@ -15,6 +15,14 @@ export function isValidTag(tag: Tag): boolean {
   return true;
 }
 
+export function isFeeRecipientEditable(tag: Tag): boolean {
```

Comment:
Again this would be more simple if you use
```
export function isFeeRecipientNonEditable(tag: Tag): boolean {
   return nonEditableFeeRecipientTags.includes(tag)
}
```